src/users/containers/invite-user/invite-user.component.ts

```diff
@@ -220,8 +220,8 @@ export class InviteUserComponent implements OnInit, OnDestroy {
         firstName: [(this.f.firstName.errors && this.f.firstName.errors.required)],
         lastName: [(this.f.lastName.errors && this.f.lastName.errors.required)],
         email: [
-          (this.f.email.errors && this.f.email.errors.required),
-          (this.f.email.errors && this.f.email.errors.email),
+          (this.f.email.errors && this.f.email.errors.required) ||
```

Comment:
This would be a little more efficient:
`this.f.email.errors && (this.f.email.errors.required || this.f.email.errors.email)`